src/clj/game/cards/operations.clj

```diff
@@ -1191,6 +1191,29 @@
     :msg "shuffle a card from HQ into R&D"
     :effect (final-effect (move target :deck) (shuffle! :deck))}
 
+   "Riot Suppression"
+   {:req (req (last-turn? state :runner :trashed-card))
+    :delayed-completion true
+    :effect (req (show-wait-prompt state :corp "Runner to decide if they will take 1 brain damage")
+                 (when-completed (resolve-ability
+                                   state :runner
```

Comment:
It's probably overkill to use when-completed here. You can just flow into a continue-ability that runs the new prompt, and then use an end-ability to clear the prompt and RFG the operation.